docs/guide/deployer.md

```diff
@@ -63,6 +89,37 @@ The main difference between deploying an ASA and SSC is that ASA takes `asset-na
 You can learn more about the flags from [Deployer API](https://scale-it.github.io/algo-builder/);
 You can learn more about Stateful Smart Contracts [here](https://developer.algorand.org/docs/features/asc1/stateful/).
 
+#### OptIn to SSC
+
+For opting in to SSC, `deployer` supports two methods :-
+- `optInAcountToSSC` to opt-in to a single account signed by secret key of sender.
+- `optInLsigToSSC` to opt-in to a contract account (say escrow) where the account is represented by the logic signature address (`lsig.address()`).
+  - To opt in to SSC you can use `Application Index`.
```

Comment:
can you add a link or explain it further?

Reply:
yes